### PR TITLE
Des 23: Add art elements

### PR DESCRIPTION
**Pull request by aspencer116**
- Add a new component that supports rendering three different SVG "art elements" which are decorative additions to the content.
- Adds these new art elements to the page according to the design with some changes based on how content flows on the real page.
- Adjusts some of the grid layout on the page.

### CODE REVIEW

File: src/components/2021/art-element.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    </svg>
+  )
+}
+
+ArtElement.propTypes = {
+  type: PropTypes.string,
+  isHero: PropTypes.bool
+}
+
+export default ArtElement
```
Comment (colbyallenc) [inline src/components/2021/art-element.js:38]: certified developer.

File: src/scss/2021/components/_art-element.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  &--hero {
+    position: absolute;
+    right: -$fixed-columns-lg - 0.125rem;
+    bottom: 6rem;
+    width: calc((100% / 3) + #{$fixed-columns-lg});
+  }
+
+  &--bars {
+    
+  }
```
Comment (colbyallenc) [inline src/scss/2021/components/_art-element.scss:20]: should we remove this?

File: src/sections/2021/section-4.js [hunk 3]
```diff
@@ -128,6 +133,10 @@ const Section4 = () => (
         </BarChart>
       </Figure>
     </GridCell>
+
+    <GridCell align="center" startMD="8" startLG="12">
+        <ArtElement type="bars" />
+      </GridCell>
```
Comment (colbyallenc) [inline src/sections/2021/section-4.js:139]: I think that the indentation here may be off just a little.

File: src/sections/2021/section-5.js [hunk 4]
```diff
@@ -99,6 +104,10 @@ const Section5 = () => (
       </Figure>
     </GridCell>
 
+    <GridCell align="center" startMD="8" startLG="12">
+        <ArtElement type="bars" />
+      </GridCell>
```
Comment (colbyallenc) [inline src/sections/2021/section-5.js:109]: indentation here as well :)

File: src/sections/2021/section-5.js [hunk 5]
```diff
@@ -140,6 +149,10 @@ const Section5 = () => (
         <a href="https://example.com" className="cmp-button cmp-button--inverse">Take the Survey</a>
       </CallToAction>
     </GridCell>
+
+    <GridCell align="end" startMD="7" startLG="11" className="util-display-none util-display-block@lg">
+        <ArtElement type="chart" />
+      </GridCell>
```
Comment (colbyallenc) [inline src/sections/2021/section-5.js:155]: once more!

**Comment by colbyallenc** [comment]
Some images are aligned to the edge of the page while others aren't. Do we want all of the art images against the edge or is it fine with space between the `svg` and the edge of the page?
Screen Shot 2021-07-01 at 10 18 19 AM
Screen Shot 2021-07-01 at 10 21 21 AM

Is it intended for the art image in the conclusion to be a lot bigger than the previous ones?
Screen Shot 2021-07-01 at 10 24 47 AM

File: src/scss/2021/components/_art-element.scss (hunk 2 above)
Comment (aspencer116) [reply src/scss/2021/components/_art-element.scss:20]: Oops, yep!

File: src/sections/2021/section-4.js (hunk 3 above)
Comment (aspencer116) [reply src/sections/2021/section-4.js:139]: Ah good catch

**Review by colbyallenc — APPROVED** [review]

**Pull request merged**